### DISCUSSION TITLE
What are the locked cabinets called in the scripts?

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#101] [Q&A] [answered]
What are the cabinets called that we use a code on?  
I opened "Mike Tanaka's room" and there's tons of loot in there.  I want to place a couple things more. 
Maybe stick a blueprint in there.  But I've searched dozens of items and can't find what the lockers are called in the scripts.
I see cupboards, lockers, strongboxes, but they're just in ayz or archetype. 
I cannot find the entities' spawn location in the scripts where they're interacted with.
I searched ambient and mission too since Tanaka's room is not a scripted item it's a backtrack loot item.
I even looked for something with the code 2931 and didn't see a door or other item with that code.

I'm a bit stumped here.

**Comment by MattFiler** [comment, marked as answer]
It's probably just an instance of one of the `ARCHETYPES\GAMEPLAY\KEYPAD*` composite variants?

If not, it could be utilising the `ARCHETYPES\MISC_SUBARCHETYPES\DOOR_MECHANISM` composite. I imagine it's either in the composite for that individual room, or in the one for the whole environment.

**Reply by ghramsey** [reply]
I knew I'd seen reference to "wall strongbox" somewhere and that indeed is the name.
I located it by looking for other missions where those strongboxes are mission critical.
I could not find the one in lowerhostpital, but corporatepent is a smaller, less involved map.
It's under ayz\props>medium\strongbox, but the reference to it is through wall_strongbox
and that's where the door is too.

And there it is in HAB_Airport. 
levels>sevastopol>baggage_collection_section>strongbox_1  

Question is now;  How do I find the correct cooridinates to add something to the box?
There's a whole lot of static loot in that room.  I'll keep looking for items references.
That should not be so hard now.

**Reply by MattFiler** [reply]
Could you not find the position of the strongbox and create an entity at that position?

**Reply by ghramsey** [reply]
I could extrapolate the location from the door and box location yeah.  
Since I want the items inside I had to find something else inside the box.

I located the EMP and Molotovs inside the box.  The rest must be randomized loot.
There were sensors and scrap. Neither of those show up in the same section so they're part of the randomized loot distrubution I think.
I don't know where to look but I can work from the EMP and molotov coordinates.

Thanks for the follow up.

**Reply by MattFiler** [reply]
No probs! I'm hoping to add some easier visualisation for in-world positions quite soon :)
